test/src/module/command/restoreTest.c

```diff
@@ -3392,6 +3392,109 @@ testRun(void)
 
         // Check that file was restored to full size with a partial write
         TEST_RESULT_LOG_EMPTY_OR_CONTAINS(", bi 128KB/256KB, ");
+        harnessLogLevelSet(logLevelWarn);
```

Comment:
I think it's worth moving this after `TEST_TITLE` so that it's clear which test it relates to.

Reply:
logLevelDetail is set in the beginning of the last existing test. We should set default log level at the end of the test